lib/ex_subtil_backend/workflow/step/ftp_upload.ex

```diff
@@ -26,21 +26,26 @@ defmodule ExSubtilBackend.Workflow.Step.FtpUpload do
   end
 
 
-  defp start_upload([], _workflow_id), do: {:ok, "started"}
-  defp start_upload([file | files], workflow_id) do
+  defp start_upload([], _workflow), do: {:ok, "started"}
+  defp start_upload([file | files], workflow) do
     hostname = System.get_env("AKAMAI_VIDEO_HOSTNAME") || Application.get_env(:ex_subtil_backend, :akamai_video_hostname)
     username = System.get_env("AKAMAI_VIDEO_USERNAME") || Application.get_env(:ex_subtil_backend, :akamai_video_username)
     password = System.get_env("AKAMAI_VIDEO_PASSWORD") || Application.get_env(:ex_subtil_backend, :akamai_video_password)
+    prefix = System.get_env("AKAMAI_VIDEO_PREFIX") || Application.get_env(:ex_subtil_backend, :akamai_video_prefix) || "/421959/prod/innovation/SubTil"
+
+    current_date =
+      Timex.local
+      |> Timex.format!("%Y_%m_%d__%H_%M", :strftime)
```

Comment:
So the date used to sort the generated DASHes is the upload date, but wouldn't it be more relevant to use the generation date instead?